test/cypress/integration/simplified-account/service-settings/card-payments/card-payments.cy.js

```diff
@@ -0,0 +1,119 @@
+      allow_google_pay: allowGooglePay ?? true
+    })
+  ])
+}
+
+describe('Card payments page', () => {
+  beforeEach(() => {
+    cy.task('clearStubs')
+    cy.setEncryptedCookies(USER_EXTERNAL_ID)
+  })
+
+  describe('Card payments landing page', () => {
+    describe('for an admin user', () => {
+      it('should show the correct heading and title', () => {
+        setupStubs()
+        cy.visit(pageUrl)
+
+        cy.get('h1').should('contain.text', 'Card payments')
+        cy.title().should('eq', 'Settings - Card payments - GOV.UK Pay')
+      })
+
+      it('should display the provided card payment details (version 1 - everything on)', () => {
+        setupStubs({
+          collectBillingAddress: true,
+          defaultBillingAddressCountry: 'GB',
+          allowApplePay: true,
+          allowGooglePay: true,
+          serviceName: 'version 1'
+        })
+        cy.visit(pageUrl)
+        cy.get('.govuk-summary-list').eq(0).within(() => {
+          cy.get('.govuk-summary-list__row').eq(0).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Collect billing address')
+            cy.get('.govuk-summary-list__value').should('contain', 'On')
+          })
+          cy.get('.govuk-summary-list__row').eq(1).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Default billing address country')
+            cy.get('.govuk-summary-list__value').should('contain', 'United Kingdom')
+          })
+        })
+        cy.get('.govuk-summary-list').eq(1).within(() => {
+          cy.get('.govuk-summary-list__row').eq(0).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Apple Pay')
+            cy.get('.govuk-summary-list__value').should('contain', 'On')
+          })
+          cy.get('.govuk-summary-list__row').eq(1).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Google Pay')
+            cy.get('.govuk-summary-list__value').should('contain', 'On')
+          })
+        })
+      })
+
+      it('should display the provided card payment details (version 2 -everything off)', () => {
+        setupStubs({
+          collectBillingAddress: false,
+          defaultBillingAddressCountry: 'IE',
+          allowApplePay: false,
+          allowGooglePay: false,
+          serviceName: 'version 2'
+        })
+
+        cy.visit(pageUrl)
+        cy.get('.govuk-summary-list').eq(0).within(() => {
+          cy.get('.govuk-summary-list__row').eq(0).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Collect billing address')
+            cy.get('.govuk-summary-list__value').should('contain', 'Off')
+          })
+          cy.get('.govuk-summary-list__row').eq(1).within(() => {
+            cy.get('.govuk-summary-list__key').should('contain', 'Default billing address country')
+            cy.get('.govuk-summary-list__value').should('contain', 'IE')
```

Comment:
Eww why is "IE" (Ireland?) here but "United Kingdom" in version 1 of the test?

Reply:
I took it from [the existing implementation](https://github.com/alphagov/pay-selfservice/blob/8d90726/app/views/settings/index.njk#L128) (because the ticket talks about the page contents not changing).  The existing behaviour is that 'GB' is replaced with 'United Kingdom' but everything else is left unedited.

I think this is worth looking into as `GB` and `UK` are not synonymous but we're treating them as though they are.